seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/config/SinkConfig.java

```diff
@@ -90,6 +92,8 @@ public static SinkConfig of(ReadonlyConfig config) {
                 .ifPresent(sinkConfig::setColumnSeparator);
         sinkConfig.setLoadFormat(config.get(StarRocksSinkOptions.LOAD_FORMAT));
         sinkConfig.setDataSaveMode(config.get(StarRocksSinkOptions.SAVE_MODE));
+        config.getOptional(StarRocksSinkOptions.HTTP_SOCKET_TIMEOUT_MS)
+                .ifPresent(sinkConfig::setHttpSocketTimeout);
```

Comment:
`HTTP_SOCKET_TIMEOUT_MS` has default value. So I think you should use `config.get(StarRocksSinkOptions.HTTP_SOCKET_TIMEOUT_MS)` instead of `config.getOptional(StarRocksSinkOptions.HTTP_SOCKET_TIMEOUT_MS)`. It always return a value used to `setHttpSocketTimeout`

Reply:
done.